### PR TITLE
Add workspace

### PR DESCRIPTION
**Pull request by djino97**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/854?issue_count=5&issue_position=1&next_issue_id=852

### CODE REVIEW

File: src/GUI/Pages/Messages/CreateWorkspace.razor [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+@using Services.Interfaces
+@using LT.DigitalOffice.GUI.Shared
+
+@inject IMessageService _MessageService;
+
+<div class=@(IsOpenCreateWorkspaceWindow ? "create-workspace-window" : "create-workspace-window-hide")>
+    <p>Create workspace</p>
+    <hr>
+
+    <label for="name">Name</label>
```
Comment (MariyaSin) [inline src/GUI/Pages/Messages/CreateWorkspace.razor]: Why do you not add Nane to resx?

File: src/GUI/Pages/Messages/CreateWorkspace.razor [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <input id="name" @bind=@_request.Name type="text" class="form-control" aria-describedby="basic-addon1" />
+
+    <label for="description">Description</label>
```
Comment (MariyaSin) [inline src/GUI/Pages/Messages/CreateWorkspace.razor]: resx?

File: src/GUI/Pages/Messages/CreateWorkspace.razor [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    <textarea rows="3" @bind=@_request.Description id="description" type="text" class="form-control"
+        aria-describedby="basic-addon1"></textarea>
+
+    <label for="image">Image</label>
+
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#snackbar-modal"
+        @onclick="CreateWorkspaceAsync">
+        Finish
```
Comment (MariyaSin) [inline src/GUI/Pages/Messages/CreateWorkspace.razor]: resx?

File: src/GUI/Pages/Messages/CreateWorkspace.razor [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    private WorkspaceRequest _request { get; set; }
+
+    protected override void OnInitialized()
+    {
+        _request = new();
+    }
+
+    public async Task CreateWorkspaceAsync()
+    {
+
```
Comment (MariyaSin) [inline src/GUI/Pages/Messages/CreateWorkspace.razor]: remove empy line

**Review by MariyaSin — APPROVED** [review]

File: src/GUI/Pages/Messages/CreateWorkspace.razor (hunk 1 above)
Comment (djino97) [reply src/GUI/Pages/Messages/CreateWorkspace.razor]: Fixed

File: src/GUI/Pages/Messages/CreateWorkspace.razor (hunk 2 above)
Comment (djino97) [reply src/GUI/Pages/Messages/CreateWorkspace.razor]: Fixed

File: src/GUI/Pages/Messages/CreateWorkspace.razor (hunk 3 above)
Comment (djino97) [reply src/GUI/Pages/Messages/CreateWorkspace.razor]: Fixed

File: src/GUI/Pages/Messages/CreateWorkspace.razor (hunk 4 above)
Comment (djino97) [reply src/GUI/Pages/Messages/CreateWorkspace.razor]: Fixed

File: src/GUI/Helpers/CreateEntityHelper.cs [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+{
+    public static class CreateEntityHelper
+    {
+        public static async Task<string> CreatioHandlerEntity<TReq, TRes>(
+            Func<TReq, Task<TRes>> request, 
+            TReq body,
+            string entityName)
+        {
+            string messageUser = "";
+            bool isSuccessOperation = false;
```
Comment (BelehovEgor) [inline src/GUI/Helpers/CreateEntityHelper.cs]: is never used

File: src/GUI/Helpers/CreateEntityHelper.cs [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+            try
+            {
+                var response = await request(body);
```
Comment (BelehovEgor) [inline src/GUI/Helpers/CreateEntityHelper.cs]: don't use var were we can't understand what type is it

**Review by BelehovEgor — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Pull request merged**